org.upbge.UPBGE.yml

```diff
@@ -0,0 +1,137 @@
+id: org.upbge.UPBGE
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: blender
+finish-args:
+  - "--share=ipc"
+  - "--socket=wayland"
+  - "--socket=fallback-x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--share=network"
+  - "--filesystem=host"
+  - "--filesystem=/run/spnav.sock:ro"
+  - "--env=SPNAV_SOCKET=/run/spnav.sock"
+  - "--env=TMP_DIR=/tmp"
+  - "--env=TMP=/tmp"
```

Comment:
host does not give access to real `/tmp` outside the sandbox. Why do you need it? Also are these actual environment variables? You can set TMPDIR to something inside the sandbox like $XDG_CACHE_HOME/tmp, XDG_RUNTIME_DIR/tmp etc.

Reply:
I wouldn't know, these permissions are taken from the Blender Flatpak to maintain consistency.

Reply:
At least the TMP_DIR env var is correct, after grepping through blender.